performance page: remove aurora comparison

awoods187 opened the pull request:
We are removing the comparison vs. aurora from the performance page.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/docs/5842)
<!-- Reviewable:end -->

cockroach-teamcity commented:
Online preview: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/1a87f91e49ffb7c228cb916e7c489430af2d90ef/

Edited pages:
- [v19.2/performance.md](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/1a87f91e49ffb7c228cb916e7c489430af2d90ef/v19.2/performance.html)

irfansharif commented on v19.2/performance.md:
```diff
@@ -17,23 +17,17 @@ This document is about CockroachDB’s performance on benchmarks. For guidance o
+Max Warehouses with Max Efficiency | 100,000 Warehouses
+Max Number of Rows                 | 49.8B             
+Max Unreplicated Data              | 8TB               
+Machine type                       | 81 c5d.9xlarge            
```
Maybe pull out 81 into a separate row?

nvanbenschoten replied:
+1

awoods187 replied:
ack and done

nvanbenschoten commented on v19.2/performance.md:
```diff
@@ -17,23 +17,17 @@ This document is about CockroachDB’s performance on benchmarks. For guidance o
-
-To learn more about our comparison with Amazon Aurora, see this [blog post](https://www.cockroachlabs.com/blog/cockroachdb-2dot1-performance/).
-{% comment %}replace link with 100k blog post{% endcomment %}
+                                   | CockroachDB       
```
Does this header make sense to include anymore?

awoods187 replied:
it screws up the styling to remove it completely. I've made it Metrics/CockroachDB Results

cockroach-teamcity commented:
Online preview: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/94840e89a080fdf911fd2a37dd313d8c0a532917/

Edited pages:
- [v19.2/performance.md](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/94840e89a080fdf911fd2a37dd313d8c0a532917/v19.2/performance.html)

cockroach-teamcity commented:
Online preview: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/7b8df8261829a418bed8c5edb851b2bee6922cfb/

Edited pages:
- [v19.2/performance.md](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/7b8df8261829a418bed8c5edb851b2bee6922cfb/v19.2/performance.html)

awoods187 commented:
We have those details but it would be a bit confusing on the machine types and nodes because we switched to using AWS and we switched CPU sizes. We used over hundred gcp nodes on 50 k but only 81 larger aws nodes on 100k.

awoods187 commented:
I think we should merge as is to get this addressed and update with additional information in the future.

ajwerner commented:
> I think we should merge as is to get this addressed and update with additional information in the future.

I agree. We ran the 50k and 10k runs at different times on different versions. We should perhaps run them again and on 19.2 to have a reasonable table but that’s follow up work.

jseldess commented:
OK, merging this as-is. FWIW, I ran 1K and 10K on aws as part of the process of writing the repro steps, so I think I have that data. But not 50k.
